app/src/main/java/com/mashup/lastgarden/ui/scent/comment/ScentCommentViewModel.kt

```diff
@@ -20,18 +39,39 @@ class ScentCommentViewModel : ViewModel() {
     val commentDetail: LiveData<Comment>
         get() = _commentDetail
 
-    fun getCommentList() {
-        val replyList = listOf<Reply>()
-        _commentList.value =
-            listOf(
-                Comment(0, "일이삼", "2020-12-12", "sdfsdfsdffd", replyList, 112, 11, 10, true),
-                Comment(0, "일이삼", "2020-12-12", "sdfsdfsdffd", replyList, 112, 11, 10, true),
-                Comment(0, "일이삼", "2020-12-12", "sdfsdfsdffd", replyList, 112, 11, 10, false),
-                Comment(0, "일이삼", "2020-12-12", "sdfsdfsdffd", replyList, 112, 11, 10, false),
-                Comment(0, "일이삼", "2020-12-12", "sdfsdfsdffd", replyList, 112, 11, 10, true),
-                Comment(0, "일이삼", "2020-12-12", "sdfsdfsdffd", replyList, 112, 11, 10, false),
-                Comment(0, "일이삼", "2020-12-12", "sdfsdfsdffd", replyList, 112, 11, 10, true)
-            )
+    private val _commentResponse = MutableLiveData<AddCommentState>()
+    val commentResponse: LiveData<AddCommentState>
+        get() = _commentResponse
+
+    private val _emitCommentList = MutableLiveData<Unit>()
+    val emitCommentList: LiveData<Unit>
+        get() = _emitCommentList
+
+    val storyId: LiveData<Int> = savedStateHandle.getLiveData("storyId", 0)
+
+    fun setStoryId(storyId: Int) {
+        savedStateHandle["storyId"] = storyId
+    }
+
+    fun getCommentList(storyId: Int) {
+        _emitCommentList.value = Unit
+        pagingCommentList =
+            storyRepository
+                .fetchCommentList(storyId, PAGE_SIZE)
+                .cachedIn(viewModelScope)
```

Comment:
```suggestion
        pagingCommentList = storyRepository
                .fetchCommentList(storyId, PAGE_SIZE)
                .cachedIn(viewModelScope)
```
변경점은 아닌데 이렇게 해야 눈에 더 와닿는것 같아요. 정말 결합된 로직인데 괜한 엔터로 결합도가 떨어져보여요.
체이닝도 사실 이런 맥락의 장점도 있어서 최대한 연장된 코드는 불필요한 엔터 없이 작성해주시면 좋을 것 같아요

Reply:
감사합니다!!👍🏻